HomeOxygenTherapy/R4/files/HomeOxygenTherapyPrepopulation-0.1.0.cql

```diff
@@ -9,39 +9,38 @@ include FHIRHelpers version '4.0.0' called FHIRHelpers
 // with guidance from https://www.cms.gov/Outreach-and-Education/Medicare-Learning-Network-MLN/MLNProducts/Downloads/Home-Oxygen-Therapy-ICN908804.pdf
 
 
+// Code Systems
 codesystem "ICD-10-CM": 'http://hl7.org/fhir/sid/icd-10-cm'
 codesystem "LOINC": 'http://loinc.org'
 codesystem "SNOMED-CT": 'http://snomed.info/sct'
 codesystem "HCPCS": 'https://bluebutton.cms.gov/resources/codesystem/hcpcs'
 codesystem "FHIRRequestIntent": 'http://hl7.org/fhir/request-intent'
 
-//COPD_Codes
-valueset "COPD": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.6'
-
-//Bronchiectasis_Codes
-valueset "Bronchiectasis": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.3'
-
-//Hypoxemia_Codes
-valueset "Hypoxemia Disorder": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.9'
-
-//Diffuse_interstitial_lung_disease_Codes
-valueset "Interstitial Lung Disease": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.12'
-
-//Cystic_fibrosis_Codes
-valueset "Cystic Fibrosis Lung Disease": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.15'
+// Qualifying Conditions
+valueset "Home Oxygen Therapy Qualifying Conditions": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.25'
 
 // Device Categories
 valueset "Stationary Oxygen Therapy Device": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.80'
 valueset "Portable Oxygen Therapy Device": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.79'
+// Device Types
 valueset "Liquid Oxygen Therapy Device": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.77'
 valueset "Compressed Gas Oxygen Therapy Device": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.74'
 valueset "Oxygen Concentrator Therapy Device": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.78'
 
-//Pulmonary_hypertension_Codes
-code "I27.0": 'I27.0' from "ICD-10-CM"
-code "I27.2": 'I27.2' from "ICD-10-CM"
+// Laboratory Test Codes
+valueset "Laboratory Test for Home Oxygen Therapy or Respiratory Assist Device Evaluation": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.106'
+
+// Immobilization Codes
+valueset "Immobilization": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.17.4077.3.2006'
+
+// Oxygen Saturation Lab Test
+valueset "Oxygen Saturation Lab Test": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.132'
+
+// Type of Device Order Valueset
+valueset "RequestIntent": 'https://www.hl7.org/fhir/valueset-request-intent.html'
```

Comment:
If only used in questionnaire, we don't need to declare in CQL file.

Reply:
@mickohanlon23 can you pull master to your branch? Valueset-R4-request-intent.json is in master but not in your branch so the questionnaire is not using it.

Reply:
I deleted the unused Value Sets and rebased onto master.